cairo1-run/src/cairo_run.rs

```diff
@@ -1,3 +1,4 @@
+use crate::error::Error;
 use cairo_lang_casm::{
     casm, casm_extend, hints::Hint, inline::CasmContext, instructions::Instruction,
 };
```

Comment:
I don't understand the duplication problem, there is no other `cairo_run_program` in the `cairo1-run` crate

Reply:
Commonly used sructs and functions such as `cairo_run_program` are also re-exported in lib.rs to make paths shorter already

Reply:
I meant "name duplication", like `cairo1_run::cairo_run_program`, you have to write cairo_run twice. I think it can be just `run_program` or `execute_program`, something like that.

Reply:
It follows the same standard as cairo_vm & cairo_lang, I think it would be confusing to break the standard for this crate